website/docs/reference/global-configs/behavior-changes.md

```diff
@@ -152,3 +154,13 @@ on-run-start:
   - '{{ ... if flags.WHICH != 'freshness' }}'
 ```
 </File>
+
+
+### MetricFlow time spine YAML
+The flag is `False` by default.
```

Comment:
```suggestion
The `require_yaml_configuration_for_mf_time_spines` flag is configured to `False` by default.
```

Reply:
good shout! I'll stick with the original and just add the flag name since your proposal is passive voice. so: 

The `require_yaml_configuration_for_mf_time_spines` flag is set to `False` by default.